src/components/UI/GlobalStyles.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  html {
+  font-size: 62.5%;
```

Comment:
Why would you scale down the font-size globally? Is your app still accessible (i.e. readable for people you ave problems with small fonts)?

Reply:
Our head couch told us to do so, so that it is easier to work with rem.
A fellow student sent me this article about it: https://www.aleksandrhovhannisyan.com/blog/62-5-percent-font-size-trick/

Reply:
Ok, I didn't know about this, so I think it's fine :)